docs/docs/responses.mdx

```diff
@@ -231,16 +189,13 @@ You can add images to a response by providing a URL to the image under the `imag
 ### Custom Output Payloads
```

Comment:
I think I would still include something about this getting sent to the output as JSON